src/main/java/seedu/quotesify/quote/QuoteParser.java

```diff
@@ -0,0 +1,177 @@
+package seedu.quotesify.quote;
+
+import seedu.quotesify.author.Author;
+import seedu.quotesify.commands.Command;
+import seedu.quotesify.exception.QuotesifyException;
+
+import java.util.HashMap;
+
+public class QuoteParser {
```

Comment:
I found a diagram mentioning this in section 4.2.2. Firstly, if a method is static, should the class name has a prefix << class >>? Secondly, in UML Sequence Diagram, should the class name not to be underlined? Thirdly, should the activation bar that I circled in the attached image be separated into 2 activation bars?
Furthermore, I found that the font size is quite small for the readers.
![422](https://user-images.githubusercontent.com/55688719/97388400-f080a300-1912-11eb-8309-7afcbb9eb562.PNG)